webpack/JobWizard/steps/AdvancedFields/__tests__/__snapshots__/AdvancedFields.test.js.snap

```diff
@@ -186,61 +281,139 @@ exports[`AdvancedFields rendring 1`] = `
-                id="time-span-help"
-              >
-                <button
-                  aria-label="open-help-tooltip-button"
-                  className="pf-c-form__group-label-help"
-                  onClick={[Function]}
+          <NumberInput
+            formProps={
+              Object {
+                "fieldId": "time-span",
+                "label": "Time span",
+                "labelIcon": <Popover
+                  aria-label="help-text"
+                  bodyContent="Distribute execution over N seconds. If this is set and proxy batch triggering is enabled, then tasks are triggered on the smart proxy in batches of size 1."
+                  id="time-span-help"
                 >
-                  <HelpIcon
-                    color="currentColor"
-                    noVerticalAlign={true}
-                    size="sm"
-                  />
-                </button>
-              </Popover>
+                  <button
+                    aria-label="open-help-tooltip-button"
+                    className="pf-c-form__group-label-help"
+                    onClick={[Function]}
+                    type="button"
+                  >
+                    <HelpIcon
+                      color="currentColor"
+                      noVerticalAlign={true}
+                      size="sm"
+                    />
+                  </button>
+                </Popover>,
+              }
+            }
+            inputProps={
+              Object {
+                "autoComplete": "time-span",
+                "id": "time-span",
+                "min": 1,
+                "onChange": [Function],
+                "placeholder": "For example: 1, 2, 3, 4, 5...",
+                "value": "",
+              }
             }
           >
-            <div />
-          </mockConstructor>
+            <mockConstructor
+              fieldId="time-span"
+              helperTextInvalid="Has to be a number"
+              label="Time span"
+              labelIcon={
+                <Popover
+                  aria-label="help-text"
+                  bodyContent="Distribute execution over N seconds. If this is set and proxy batch triggering is enabled, then tasks are triggered on the smart proxy in batches of size 1."
+                  id="time-span-help"
+                >
+                  <button
+                    aria-label="open-help-tooltip-button"
+                    className="pf-c-form__group-label-help"
+                    onClick={[Function]}
+                    type="button"
+                  >
+                    <HelpIcon
+                      color="currentColor"
+                      noVerticalAlign={true}
+                      size="sm"
+                    />
+                  </button>
+                </Popover>
+              }
+            >
+              <div />
+            </mockConstructor>
+          </NumberInput>
```

Comment:
If you want, you can remove this snapshot file so you don't need to maintain it in other PRs as well, in continuous to the discussion in https://community.theforeman.org/t/rethinking-react-redux-folder-structure/24183/8